app/utils/communication/future_notifications.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+
+from fastapi import BackgroundTasks
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.core.notification.notification_types import CustomTopic
+from app.core.notification.schemas_notification import Message
+from app.types.scheduler import Scheduler
+from app.utils.communication.notifications import NotificationManager, NotificationTool
+
+
+class FutureNotificationTool(NotificationTool):
+    def __init__(
+        self,
+        background_tasks: BackgroundTasks,
```

Comment:
The `FutureNotificationTool` don't need to use a `BackgroundTasks`. Deferred tasks will be run in a dedicated worker